tests/Rector/ClassMethod/RemoveUnusedRequestParamRector/Fixture/controller.php.inc

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Rector\Symfony\Tests\Rector\ClassMethod\RemoveUnusedRequestParamRector\Fixture;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+abstract class AbstractController extends Controller
+{
+    abstract public function fooAction(Request $request): Response;
+}
+
+class DemoController extends AbstractController
```

Comment:
It needs to be more specific name so it ease to look on the future, eg:

```suggestion
class DontOverrideParentClassMethodController extends AbstractController
```

Reply:
++